### ISSUE TITLE
Execute command on startup?

### ISSUE THREAD

**Issue by egasimus** [fish-shell/fish-shell#1527]
It's inconvenient to modify `config.fish` for one-time commands, and `fish -c` exits once the command is done. Is there an idiomatic way of starting a command in Fish, but keeping the shell running after it's done? Even something as simple as setting the start directory with `sh -c 'cd /tmp && fish'` doesn't seem to work, and I was actually hoping I would be able to go beyond that and start a console application which exits to Fish when it's done.

**Comment by fzakaria** [comment]
I used the following

    eval jenv init - > /dev/null 2>&1

**Comment by ofek** [comment]
Any progress on this?

**Comment by mqudsi** [comment]
The example in the original post `sh -c 'cd /tmp && fish'` now works (shell does not exit, accepts input).

This can probably be closed.

**Comment by krader1961** [comment]
This is a duplicate of #4164 which has been implemented and will be in the upcoming 2.7.0 release.
